app/models/nic/base.rb

```diff
@@ -209,6 +209,10 @@ def to_audit_label
       [mac, name, identifier, _('Unnamed')].detect(&:present?)
     end
 
+    def to_s
+      identifier || mac
+    end
+
```

Comment:
This leads to much nicer logging, since this was "name" previously which is often empty log lines "Orchestrating DHCP for " (empty string - meaningless line).